app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
ooohh i see thanks @katiecrane ! And is this just to improve performance if you make it an index?